src/app.js

```diff
@@ -1,21 +1,58 @@
+/* eslint-disable no-console */
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const fs = require('fs');
+const path = require('path');
+
+function moveFile() {
+  const [enteredSourcePath, enteredDestinationPath] = process.argv.slice(2);
+  const fileName = path.basename(enteredSourcePath);
+  const basePath = path.resolve(__dirname, `../`);
+  const sourcePath = path.resolve(basePath, enteredSourcePath);
+  const destinationPath = path
+    .resolve(basePath, enteredDestinationPath);
+  const destinationStats = fs.statSync(
+    destinationPath,
+    { throwIfNoEntry: false }
+  );
+
+  if (!fs.existsSync(sourcePath)) {
+    console.info('The file does not exist!!!');
+
+    return;
+  }
+
+  if (!fs.existsSync(destinationPath) && enteredDestinationPath.endsWith('/')) {
+    console.info('Invalid destination path!!!');
+
+    return;
+  } else if (!fs.existsSync(destinationPath)) {
+    console.info('Invalid destination directory. Trying to rename file...');
+  }
+
+  if (destinationStats && destinationStats.isDirectory()) {
```

Comment:
You don't need to check `destinationStats`, because you don't handle the case if there is no `destinationStats`.

Reply:
I agree))